devtools/common/src/main/java/io/quarkus/cli/commands/file/GradleBuildFile.java

```diff
@@ -185,25 +170,25 @@ public void accept(String currentLine) {
     }
 
     @Override
-    protected boolean hasDependency(Extension extension) {
+    protected boolean hasDependency(Extension extension) throws IOException {
         return getDependencies().stream()
                 .anyMatch(d -> extension.getGroupId().equals(d.getGroupId())
                         && extension.getArtifactId().equals(d.getArtifactId()));
     }
 
     @Override
-    protected boolean containsBOM() {
-        return buildContent.contains("enforcedPlatform(\"io.quarkus:quarkus-bom:");
+    protected boolean containsBOM() throws IOException {
+        return getModel().getBuildContent().contains("enforcedPlatform(\"io.quarkus:quarkus-bom:");
```

Comment:
although `G:A:V` syntax is more common, people can also declare gradle dependencies with expanded syntax like this:
```groovy
group: 'io.quarkus', name: 'quarkus-bom', version: '0.25.0'
```
with any amount of spacing or newlines in between

Reply:
Right, not mentioning the code formatting could vary. This functionality needs to be re-written properly.

Reply:
`qute` to the rescue :)